ckanext/depositar_theme/fanstatic/scripts/alert.js

```diff
@@ -1,4 +1,4 @@
 (function() {
   let alertEl = document.getElementsByClassName('alert-dismissible');
-  alertEl.forEach(e => e.innerHTML += '<button type="button" class="close" data-dismiss="alert" aria-label="Close"><span aria-hidden="true">&times;</span></button>');
+  alertEl.forEach(e => e.innerHTML += '<button type="button" class="btn-close" data-bs-dismiss="alert" aria-label="Close"></button>');
```

Comment:
```suggestion
  alertEl.forEach(e => e.innerHTML += '<button type="button" class="btn-close close" data-bs-dismiss="alert" aria-label="Close"></button>');
```

Align with the [upstream](https://github.com/ckan/ckan/blob/ckan-2.10.7/ckan/public/base/javascript/notify.js#L54).